<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-0KBR3DP" name="tests.home.TestHomePage" tests="1" failures="1" timestamp="2021-07-26T15:27:12 IST" time="39.475" errors="0">
  <testcase name="testRegisterNewUserByUniqueEmailId" time="39.475" classname="tests.home.TestHomePage">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at utils.AssertUtil.assertTrue(AssertUtil.java:10)
at tests.home.TestHomePage.testRegisterNewUserByUniqueEmailId(TestHomePage.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testRegisterNewUserByUniqueEmailId -->
  <system-out>
    <![CDATA[STEP - Navigate to Magento Website Home Page.
STEP - Launch Browser Successfully...
STEP - Navigate to Casa39 English Website Home Page.
STEP - Accepting Cookies...
STEP - Go to Customer Register Page
Click - SIGN IN
Click - New account
STEP - Set required fields
Set - First
Set - Last
Set - email_0726152632415@test.com
Set - password@123
STEP - Create account
Click - Create an Account
STEP - Log In new username and Password
Click - SIGN IN
Click - Login
STEP - Go to My Account Section
Click - MY ACCOUNT
Verified - My Account Page is Displayed
STEP - Validate Account Information
Verified - User Name as is Correct
Verified - User Email Id as 'email_0726152632415@test.com' is Correct
STEP - Clicking CASA39 Logo to Return On Home Page...
Click - 
STEP - Validate Empty Cart and Close
Click - My Cart
â‚¬245.90
Verified - You have no items in your shopping cart.]]>
  </system-out>
</testsuite> <!-- tests.home.TestHomePage -->
